app/models/post.rb

```diff
@@ -0,0 +1,22 @@
+class Post < ApplicationRecord
+  belongs_to :author, class_name: 'User', foreign_keys: 'author_id'
+  has_many :posts, foreign_keys: 'author_id'
+  has_many :comments
+  has_many :likes
+
+  validates :author_id, presence: true
+  validates :title, presence: true
+  validates :text, presence: true
+
+  after_save :update_post_counter
+
+  private
+
+  def recent_comments
+    comments.order(created_at: :desc).limit(5)
+  end
```

Comment:
- [ ] Almost there! Just one small suggestion. It seems like you're on the right track with the `recent_comments` method, but I recommend making it a public method instead of a private one. Since It'll be used outside of the class, specifically in the upcoming milestones when creating views, making it accessible from outside the class is important.
- As a side note the `update_post_counter` method should still remain `private`. 🤓

Reply:
well noted, class changed from private to public